circom/tests/arrays/array8.circom

```diff
@@ -1,6 +1,6 @@
 pragma circom 2.0.0;
 // REQUIRES: circom
-// RUN: rm -rf %t && mkdir %t && %circom --llvm -o %t %s
+// RUN: rm -rf %t && mkdir %t && %circom --llvm -o %t %s | sed -n 's/.*Written successfully:.* \(.*\)/\1/p' | xargs cat | FileCheck %s
```

Comment:
This lit tests get more ghetto by the day but I dont think there is a cleaner solution. Wrapping it in a shell script doesnt count. It just moves the problem elsewhere